app/public/fonts/bootstrap/glyphicons-halflings-regular.svg

```diff
@@ -0,0 +1,288 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
Similar to the other pull request, remove these fonts/bootstrap files from your changes.  Also, squash or fixup the removal into the commit that added these files so no one else has to deal with a conflict between your tracked files and their untracked versions.